src/code/components/allele-filters.js

```diff
@@ -1,4 +1,4 @@
-import {PropTypes} from 'react';
+import React, {PropTypes} from 'react';
```

Comment:
Unit tests fail without the import of React, so at this point we could enable the `react-in-jsx-scope` rule if we were so inclined or figure out a way to load React globally in setup.js, analogous to the way BioLogica is handled now.